atomic_reactor/utils/koji.py

```diff
@@ -340,12 +341,20 @@ def get_image_output(workflow, image_id, arch):
     :return: tuple, (metadata dict, Output instance)
 
     """
-    saved_image = workflow.exported_image_sequence[-1].get('path')
     image_name = get_image_upload_filename(workflow.exported_image_sequence[-1],
                                            image_id, arch)
 
-    metadata = get_output_metadata(saved_image, image_name)
-    output = Output(file=open(saved_image), metadata=metadata)
+    # Provide empty tar file for docker archive metadata
+    f = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
This patch currently provides tar archive with a single empty file located in /tmp dir e.g. `tmp/tmpi9jblah`

Is it OK or do we want to provide some README file in the archive with an explanation message for user?
For example: `Brew archive does not provide image anymore to safe Red Hat storage. Archive purpose is to provide only OSBS build metadata. More information can be found in SPCLOUD-XXXX ticket.`

Reply:
I like the idea of providing a README. I wouldn't mention Jira tickets in it, maybe mention the pullspec of the built image?

Reply:
Don't mention brew, Red Hat, or anything downstream related. If readme needs customized content, it must be config option for that (I prefer not)

Reply:
yeah I'd add just general info in the README

Reply:
README file updated